librad/src/lib.rs

```diff
@@ -25,14 +25,15 @@ extern crate lazy_static;
 #[macro_use]
 extern crate radicle_macros;
 
+pub extern crate link_canonical as canonical;
```

Comment:
Actually, `pub use` would be equivalent when there's no `#[macro_use]`. Not sure, it doesn't seem like `extern crate` would be deprecated from the language anytime soon.